environment.py

```diff
@@ -533,5 +533,16 @@ def env() -> Mapping[str, Optional[str]]:
         # testing feature branches in GitLab before they are merged to the
         # develop branch, 0 otherwise. Personal deployments have this set to 0.
         #
-        'AZUL_IS_SANDBOX': '0'
+        'AZUL_IS_SANDBOX': '0',
+
+        # The Slack workspace ID where notifications from an SNS topic will be
+        # delivered. This field needs to be retrived from the AWS Chatbot console.
```

Comment:
```suggestion
        # delivered.
```

Add reference to README for details.